src/plugins/interactive_setup/server/verification_service.test.ts

```diff
@@ -8,7 +8,7 @@
 
 import fs from 'fs';
 
-import { loggingSystemMock } from '@kbn/core/server/mocks';
+import { loggingSystemMock } from '@kbn/core-logging-server-mocks';
```

Comment:
Needed to import from the low-level package as the dependency tree was conflicting with this test mocking `@kbn/utils`